src/PubSubQueue.php

```diff
@@ -165,6 +165,15 @@ public function pop($subscriber = null)
             'returnImmediately' => true,
             'maxMessages' => 1,
         ]);
+        $queue = $this->getQueue($subscriber);
+        if ($this->config && $this->config['subscribers'] && $queue && isset($this->config['subscribers'][$queue])) {
+            $deadline = $this->config['subscribers']['deadline'];
```

Comment:
@aman00323 It must be $this->config['subscribers'][$queue]['deadline'] . correct? Please add sample config in readme